.github/workflows/ci.yaml

```diff
@@ -20,6 +23,10 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v1
 
+      - name: Remove existing rust toolchain (Windows)
+        if: matrix.os == 'windows-latest'
+        run: Rename-Item C:\Users\runneradmin\.rustup\toolchains\stable-x86_64-pc-windows-msvc C:\Users\runneradmin\.rustup\toolchains\stable-x86_64-pc-windows-msvc.old
```

Comment:
Maybe add a comment that is prevents installing `rust-docs`, which is slow, while not having to delete it, which is slow too?

Reply:
Good idea.